src/platform/utilities/tests/ui/asyncLoader.unit.spec.jsx

```diff
@@ -1,45 +1,41 @@
 import React from 'react';
 import { expect } from 'chai';
-import { findDOMNode } from 'react-dom';
-import ReactTestUtils from 'react-dom/test-utils';
+import { shallow } from 'enzyme';
 
 import asyncLoader from '../../ui/asyncLoader';
 
 describe('asyncLoader', () => {
-  // it('should display loading indicator while waiting', () => {
-  //   const Component = asyncLoader(() => new Promise(f => f), 'Test loading');
+  it('should display loading indicator while waiting', () => {
+    const Component = asyncLoader(() => new Promise(f => f), 'Test loading');
 
-  //   const page = ReactTestUtils.renderIntoDocument(<Component />);
-  //   const pageDOM = findDOMNode(page);
+    const wrapper = shallow(<Component />);
```

Comment:
Interesting that you went to enzyme for this test. I thought we were ditching enzyme in favor of RTL?

Reply:
@adamwhitlock1 I'll give it a try rewriting into RTL, I'm trying to remember the exact reason why I didn't again.

Reply:
I updated the `asyncLoader.unit.spec.jsx` to using RTL as you suggested. Thank you @adamwhitlock1 for the catch here.